src/EnergyPlus/BoilerSteam.hh

```diff
@@ -141,19 +142,14 @@ namespace BoilerSteam {
 
         void getDesignCapacities(const PlantLocation &EP_UNUSED(calledFromLocation), Real64 &MaxLoad, Real64 &MinLoad, Real64 &OptLoad) override;
 
-        void getSizingFactor(Real64 &SizFac) override;
+        void getSizingFactor(Real64 &sizFac) override;
 
         void onInitLoopEquip(EnergyPlusData &EP_UNUSED(state), const PlantLocation &EP_UNUSED(calledFromLocation)) override;
 
-        static PlantComponent *factory(std::string const &objectName);
+        static PlantComponent *factory(BoilerSteamData &boilers, std::string const &objectName);
```

Comment:
Boiler and chiller data classes now being passed into the factory and GetInput methods.